docs/config.md

```diff
@@ -188,8 +188,8 @@ Important environment variables include:
 
 | Environment variable                           | Description                                                                                                                                                                                                                                                                                                                        | Default value |
 |------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
-| `OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP`      | Whether the tracer is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Tracer on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations.                                                               | `true`        |
-| `OTEL_DOTNET_AUTO_LOAD_METER_AT_STARTUP`       | Whether the meter is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Meter on its own.                                                                                                                                                         | `true`        |
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED`              | Whether the tracer is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Tracer on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations.                                                               | `true`        |
```

Comment:
I think it would be better if just say here that it disables/enables signal providers and source instrumentations. We could add a footer and explain when this could be handy